lib/order.rb

```diff
@@ -0,0 +1,50 @@
+class Order
```

Comment:
This is a common problem in OOD - which classes should be responsible for what?  If you end up with responsibilities in the wrong places, you can have spaghetti code and lots of interdependent classes.

Should order also be responsible for calculating the price and placing itself?  Probably not.  You place the order with the takeaway.  The order is just the list of items and quantities.  Where you are using an array or hash to store a collection of related properties, you probably need a class (eg. `OrderLine`